### ISSUE TITLE
EmptyBatchRequest when stopping and no messages

### ISSUE THREAD

**Issue by sneljo1** [PruvoNet/squiss-ts#167]
## Expected Behavior

Process should shut down silently whether messages need to be deleted or not.

## Current Behavior

Throws EmptyBatchRequest when shutting down.

```
EmptyBatchRequest: There should be at least one DeleteMessageBatchRequestEntry in the request.
  File "/app/packages/private-utils/node_modules/@aws-sdk/client-sqs/dist-cjs/protocols/Aws_json1_0.js", line 1366, col 23, in de_EmptyBatchRequestRes
    const exception = new models_0_1.EmptyBatchRequest({
  File "/app/packages/private-utils/node_modules/@aws-sdk/client-sqs/dist-cjs/protocols/Aws_json1_0.js", line 500, col 25, in de_DeleteMessageBatchCommandError
    throw await de_EmptyBatchRequestRes(parsedOutput, context);
  File "node:internal/process/task_queues", line 96, col 5, in processTicksAndRejections
  File "/app/packages/private-utils/node_modules/@smithy/middleware-serde/dist-cjs/deserializerMiddleware.js", line 7, col 24, in <anonymous>
    const parsed = await deserializer(response, options);
  File "/app/packages/private-utils/node_modules/@aws-sdk/client-sqs/node_modules/@aws-sdk/middleware-signing/dist-cjs/awsAuthMiddleware.js", line 14, col 20, in <anonymous>
    const output = await next({
  File "/app/packages/private-utils/node_modules/@smithy/middleware-retry/dist-cjs/retryMiddleware.js", line 27, col 46, in <anonymous>
    const { response, output } = await next(args);
  File "/app/packages/private-utils/node_modules/@aws-sdk/client-sqs/node_modules/@aws-sdk/middleware-logger/dist-cjs/loggerMiddleware.js", line 7, col 26, in <anonymous>
    const response = await next(args);
  File "/app/packages/private-utils/node_modules/squiss-ts/dist/Squiss.js", line 467, col 9, in Squiss._deleteXMessages
    await this._deleteMessages(delBatch);
  File "/app/packages/private-utils/node_modules/squiss-ts/dist/Squiss.js", line 312, col 9, in Squiss._drainDeleteQueue
    await this._deleteXMessages();
  File "/app/packages/private-utils/node_modules/squiss-ts/dist/Squiss.js", line 277, col 13, in Squiss.stop
    await this._drainDeleteQueue();
```
## Possible Solution

Do not make any calls to AWS if delQueue is empty

## Steps to Reproduce (for bugs)

The issue triggers for me when I didn't receive any messages, can be reproduced like so:
```
const squiss = new Squiss({
  awsConfig,
  queueUrl:"url",
  bodyFormat: 'json',
  maxInFlight: 1
});

squiss.start();
squiss.stop()

```

## Your Environment

* Version used: v5.2.3
* Node version: v16.20.0

**Comment by regevbr** [comment]
@sneljo1 thanks for reporting, fixed in 5.3.0

**Comment by sneljo1** [comment]
Awesome thanks, great turn-around time 👏
